pset-functions.js

```diff
@@ -91,6 +116,30 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount(phrase) {
+  const array = phrase.split(" ").map(x => x.replace(/,/g,''))
+  const uniqueValues = [...new Set(array)]
+  const result = {}
+
+  for (let i = 0; i < uniqueValues.length; i++) {
```

Comment:
@jacobkelman interesting approach, however, you generally want to avoid nesting `for` loops or `if` statements if possible - check out an [alternative approach here](https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L107)